application-workloads/sharepoint/sharepoint-adfs/README.md

```diff
@@ -25,24 +27,40 @@ This template deploys SharePoint 2019, 2016 or 2013 with the following configura
 * Latest version of claims provider [LDAPCP](https://ldapcp.com/) is installed and configured.
 * A 2nd SharePoint server can optionally be added to the farm.
 
-All subnets are protected by a Network Security Group with rules that restrict network access. You can connect to virtual machines using:
+## Remote access and security
+
+The template creates 1 virtual network with 3 subnets. All subnets are protected by a [Network Security Group](https://docs.microsoft.com/en-us/azure/virtual-network/network-security-groups-overview) with no custom rule by default.
+
+The following parameters impact the remote access of the virtual machines, and the network security:
+
+* Parameter 'addPublicIPAddressToEachVM':
+  * if true (default value): Each virtual machine gets a public IP, a DNS name, and may be reachable from Internet.
+  * if false: No public IP resource is created.
+* Parameter 'RDPTrafficAllowed':
+  * If 'No' (default value): Firewall denies all incoming RDP traffic from Internet.
+  * If '*' or 'Internet': Firewall accepts all incoming RDP traffic from Internet.
+  * If 'ServiceTagName': Firewall accepts all incoming RDP traffic from the specified 'ServiceTagName'.
+  * If 'xx.xx.xx.xx': Firewall accepts incoming RDP traffic only from the IP 'xx.xx.xx.xx'.
+* Parameter 'addAzureBastion':
+  * if true: Configure service [Azure Bastion](https://azure.microsoft.com/en-us/services/azure-bastion/) to allow a secure remote access.
+  * if false (default value): Service Azure Bastion is not created.
+
+## Cost
 
-* [Azure Bastion](https://azure.microsoft.com/en-us/services/azure-bastion/) if you set parameter addAzureBastion to 'Yes'.
-* RDP protocol if you set parameter addPublicIPToVMs to 'Yes'. Each machine will have a public IP, a DNS name, and the TCP port 3389 will be allowed from Internet.
+By default, virtual machines use [B-series burstable](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-b-series-burstable), ideal for such template and much cheaper than other comparable series.  
+Below is the default size and storage type per virtual machine role. Prices are shown in US Dollar, per month, as of 2021-09-13, in region West Europe, without enabling the '[Azure Hybrid Benefit](https://azure.microsoft.com/en-us/pricing/hybrid-benefit/)' licensing benefit, assuming they run 24*7:
 
-By default, virtual machines use standard storage and are sized with a good balance between cost and performance:
+* DC: Size [Standard_B2s](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes-b-series-burstable) (2 vCPU / 4 GiB RAM) ($40.88) and OS disk is a 128 GiB standard HDD ($5.89).
```

Comment:
remove the pricing information from here - it changes and can be different for different customers